bin/maintenance/make-release.py

```diff
@@ -257,11 +258,13 @@ def _update_security_md(new_version: str, dry_run: bool = False):
         else f'🚑 Limited (until {deadline_txt})'
     )
 
-    new_table = f'''| Version | Supported |
-| ------- | --------- |
-| {target_minor_version}.x | ✅ Yes (latest version) |
-| {prev_minor_release}.x | {support_text} |
-| Others | ❌ No |'''
+    new_table = textwrap.dedent(f'''
+        | Version | Supported |
+        | ------- | --------- |
+        | {target_minor_version}.x | ✅ Yes (latest version) |
+        | {prev_minor_release}.x | {support_text} |
+        | Others | ❌ No |
+    ''').strip()
```

Comment:
Really? :angry: :laughing:

Reply:
(anyway, let me be clear that I'm joking, its a perfectly valid choice, although my preferred one :laughing:)

Reply:
you really like lines starting before the surrounding function's indentation? :P